package.json

```diff
@@ -119,7 +119,9 @@
         {
           "locales": [
             "en",
-            "es"
+            "es",
+            "ko",
+            "zh"
```

Comment:
Just noticed this, were this meant to be changed and related to the class syntax for this PR?